nixos/modules/config/users-groups.nix

```diff
@@ -635,9 +635,9 @@ in {
              || cfg.passwordFile != null
              || cfg.openssh.authorizedKeys.keys != []
              || cfg.openssh.authorizedKeys.keyFiles != [])
-          ) cfg.users) ++ [
+          ) cfg.users ++ [
```

Comment:
According to https://nixos.org/manual/nix/stable/expressions/language-operators.html concat has lower precedence than func application so this is correct